rpki-validator/src/main/java/net/ripe/rpki/validator3/api/roaprefixassertions/RoaPrefixAssertionsController.java

```diff
@@ -109,7 +112,11 @@
         return ResponseEntity.ok(ApiResponse.data(toResource(roaPrefixAssertionsService.get(id))));
     }
 
-    @PostMapping(consumes = { Api.API_MIME_TYPE, "application/json" })
+    @ApiOperation(value = "Add whitelist entry", notes =
+            "By adding a whitelist entry you can manually authorise an ASN to originate a prefix in addition to the validated ROAs from the repository.\n" +
+            "Please note that whitelist entries may invalidate announcements for this prefix from other ASNs, just like ROAs. This may be intentional\n" +
+            "(you are whitelisting ASN A, ASN B is hijacking), or not (ASN B should also be authorised, or you made a mistake).")
```

Comment:
"you are whitelisting ASN A if ASN B is hijacking"? Or that's exactly how you wanted it to be?

Reply:
This was not clear (but was taken from the web interface). I tried to clarify it.

Reply:
This was not clear (but was taken from the web interface). I tried to clarify it.